src/main/kotlin/no/nav/familie/tilbake/service/dokumentbestilling/handlebars/dto/BaseDokument.kt

```diff
@@ -0,0 +1,55 @@
+
+package no.nav.familie.tilbake.service.dokumentbestilling.handlebars.dto
+
+import no.nav.familie.kontrakter.felles.tilbakekreving.Språkkode
+import no.nav.familie.kontrakter.felles.tilbakekreving.Ytelsestype
+
+open class BaseDokument(open val ytelsestype: Ytelsestype,
+                        open val språkkode: Språkkode,
+                        open val behandlendeEnhetsNavn: String,
+                        open val ansvarligSaksbehandler: String ) {
+
+    private val infoMap =
+            mapOf(Ytelsestype.BARNETRYGD to Ytelsesinfo("nav.no/barnetrygd",
+                                                        mapOf(Språkkode.NB to Ytelsesnavn("barnetrygd", "barnetrygden"),
```

Comment:
Er usikker på om me bør kopiere dette frå fptilbake. Er usikker på kor mange plasser den faktisk hentet bokmålsteksten og nynorskteksten frå her?
Hadde eigentleg eit ynske om å flytte denne til handlebar-templatane.

Reply:
Sjekket fptilbake. Dei brukes i Henleggelsesbrevet og "Innhent dokumentasjon"-brevet.

Reply:
Meiner å huske at grunnen var eit par situasjonar kor ytelsesnavnBestemt/utelsesnanvUbestemt var avhengig av situasjonen og ytelsen.
Og eg måtte eventuelt ha lagt til ein tredje status for å støtte det. Eg kom då til konklusjonen at det var meir riktig å håndtere det i handlebars.

Reply:
Jeg konkluderte motsatt, fordi man ender opp med å skrive samme koden i mange templates i stedet for å ha den ett sted.